js/components/controlBar.js

```diff
@@ -506,25 +498,21 @@ var ControlBar = React.createClass({
 
       "quality": (function (alignment) {
         return <div className="oo-popover-button-container" key="quality">
-          <button
-            type="button"
-            ref={function(e) { this.qualityBtnElement = e }.bind(this)}
-            className={qualityClass}
+          <AccessibleButton
+            ref={function(e) { this.toggleButtons[CONSTANTS.MENU_OPTIONS.VIDEO_QUALITY] = e }.bind(this)}
             style={selectedStyle}
-            onMouseUp={Utils.blurOnMouseUp}
-            onClick={this.handleQualityClick}
-            onKeyDown={this.handleQualityKeyDown}
-            data-focus-id="quality"
-            tabIndex="0"
-            aria-label={CONSTANTS.ARIA_LABELS.VIDEO_QUALITY}
-            aria-haspopup="true"
-            aria-expanded={this.props.controller.state.videoQualityOptions.showVideoQualityPopover ? true : null}>
+            className={qualityClass}
+            focusId="quality"
+            ariaLabel={CONSTANTS.ARIA_LABELS.VIDEO_QUALITY}
+            ariaHasPopup="true"
+            ariaExpanded={this.props.controller.state.videoQualityOptions.showPopover ? true : null}
```

Comment:
How come we are using null instead of false here?

Reply:
For some of these attribute it's better not to render them unless they are true or have a value. React doesn't render attributes that are set to `null`